subsys/cpool/src/main/java/org/commonjava/indy/subsys/cpool/CPInitialContextFactory.java

```diff
@@ -0,0 +1,132 @@
+package org.commonjava.indy.subsys.cpool;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.Name;
+import javax.naming.NamingException;
+import javax.naming.spi.InitialContextFactory;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.Map;
+
+public class CPInitialContextFactory
+        implements InitialContextFactory
+{
+    public static final String FACTORY_SYSPROP = "java.naming.factory.initial";
```

Comment:
just nitpick. Suggest to use javax.naming.Context.INITIAL_CONTEXT_FACTORY  instead of this literal string const